source/features/useful-forks.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'dom-chef';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+import {RepoForkedIcon} from '@primer/octicons-react';
+
+import features from '.';
+import {getRepo} from '../github-helpers';
+import looseParseInt from '../helpers/loose-parse-int';
+
+async function init(): Promise<void | false> {
+	const forkCount = looseParseInt((await elementReady('.social-count[href$="/network/members"]', {waitForChildren: false}))!);
```

Comment:
The children in this case is a 2-character text node. Not important. Also, too many parens
```suggestion
	const forkCount = looseParseInt(await elementReady('.social-count[href$="/network/members"]')!);
```

Reply:
I think it needs the parens for lint

Reply:
Fixed.